docs/development/dev_howto.rst

```diff
@@ -569,12 +569,21 @@ As explained in the
 of the Astropy docs, there are (at least) two approaches for adding to the releases,
 each with pros and cons.
 
-We've had some pain due to merge conflicts in the releases notes and having to wait
-until the contributor rebases (and having to explain git rebase to new contributors).
+We recently chose to utilise `towncrier <https://towncrier.readthedocs.io/en/stable/>`__ for our release notes.
+
+- For each PR, a related 'fragment' file should be created in the ``docs/release-notes`` folder.
+- The naming convention of the file should be ``<PULL REQUEST NUMBER>.<TYPE>.rst``, where the available
+  types are ``infrastructure``, ``docs``, ``feature`` or ``bugfix``.
+- The file should contain a suitable message for the PR, for example "Add a new feature for the estimators.".
+
+To build the release notes you can simply utilise the following line:
+
+.. code-block:: bash
+
+    towncrier build --version=<VERSION NUMBER>
+
```

Comment:
I suppose this will require an additional step during the release process as well.